Core/GDCore/Project/VariablesContainersList.h

```diff
@@ -115,11 +115,23 @@ class GD_CORE_API VariablesContainersList {
   }
 
   /**
-   * Get the variables container for a given variable.
+   * Get the variables container for a given variable or property or parameter.
    */
   const VariablesContainer &
   GetVariablesContainerFromVariableName(const gd::String &variableName) const;
 
+  /**
+   * Get the variables container for a given variable or property.
+   */
+  const VariablesContainer &
+  GetVariablesContainerFromVariableNameExcludingParameters(const gd::String &variableName) const;
+
+  /**
+   * Get the variables container for a given variable.
+   */
+  const VariablesContainer &
+  GetVariablesContainerFromVariableNameExcludingParametersAndProperties(const gd::String &variableName) const;
```

Comment:
Could have been worth naming things in the other way:

FromVariableOrPropertyOrParameterName
FromVariableOrPropertyNameOnly
FromVariableNameOnly

to mimick the parameter type names